[2024-06-14T11:51:17.681+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: simple_spark_dag.spark_submit_task manual__2024-06-14T11:51:13.610326+00:00 [queued]>
[2024-06-14T11:51:17.693+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: simple_spark_dag.spark_submit_task manual__2024-06-14T11:51:13.610326+00:00 [queued]>
[2024-06-14T11:51:17.694+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2024-06-14T11:51:17.714+0000] {taskinstance.py:1380} INFO - Executing <Task(SparkSubmitOperator): spark_submit_task> on 2024-06-14 11:51:13.610326+00:00
[2024-06-14T11:51:17.744+0000] {standard_task_runner.py:57} INFO - Started process 9741 to run task
[2024-06-14T11:51:17.778+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'simple_spark_dag', 'spark_submit_task', 'manual__2024-06-14T11:51:13.610326+00:00', '--job-id', '69', '--raw', '--subdir', 'DAGS_FOLDER/simple_spark_dag.py', '--cfg-path', '/tmp/tmprpn1zw2o']
[2024-06-14T11:51:17.821+0000] {standard_task_runner.py:85} INFO - Job 69: Subtask spark_submit_task
[2024-06-14T11:51:18.642+0000] {task_command.py:415} INFO - Running <TaskInstance: simple_spark_dag.spark_submit_task manual__2024-06-14T11:51:13.610326+00:00 [running]> on host 963b3fbfa4c9
[2024-06-14T11:51:20.483+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='simple_spark_dag' AIRFLOW_CTX_TASK_ID='spark_submit_task' AIRFLOW_CTX_EXECUTION_DATE='2024-06-14T11:51:13.610326+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-14T11:51:13.610326+00:00'
[2024-06-14T11:51:20.640+0000] {base.py:73} INFO - Using connection ID 'Spark_connection' for task execution.
[2024-06-14T11:51:20.646+0000] {spark_submit.py:401} INFO - Spark-Submit cmd: spark-submit --master spark://spark-master:7077 --conf spark.hadoop.fs.s3a.endpoint=http://host.docker.internal:9000 --conf spark.hadoop.fs.s3a.access.key=khaiVDT2024 --conf spark.hadoop.fs.s3a.secret.key=****** --conf spark.hadoop.fs.s3a.path.style.access=true --name spark_minio_job --deploy-mode client jobs/python/test2.py
[2024-06-14T11:51:21.389+0000] {spark_submit.py:571} INFO - /home/***/.local/lib/python3.9/site-packages/pyspark/bin/load-spark-env.sh: line 68: ps: command not found
[2024-06-14T11:51:27.045+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:27 INFO SparkContext: Running Spark version 3.4.0
[2024-06-14T11:51:27.140+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:27 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2024-06-14T11:51:27.255+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:27 INFO ResourceUtils: ==============================================================
[2024-06-14T11:51:27.260+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:27 INFO ResourceUtils: No custom resources configured for spark.driver.
[2024-06-14T11:51:27.262+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:27 INFO ResourceUtils: ==============================================================
[2024-06-14T11:51:27.264+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:27 INFO SparkContext: Submitted application: Spark MinIO Example
[2024-06-14T11:51:27.285+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:27 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2024-06-14T11:51:27.302+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:27 INFO ResourceProfile: Limiting resource is cpu
[2024-06-14T11:51:27.304+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:27 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2024-06-14T11:51:27.397+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:27 INFO SecurityManager: Changing view acls to: ***
[2024-06-14T11:51:27.400+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:27 INFO SecurityManager: Changing modify acls to: ***
[2024-06-14T11:51:27.402+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:27 INFO SecurityManager: Changing view acls groups to:
[2024-06-14T11:51:27.403+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:27 INFO SecurityManager: Changing modify acls groups to:
[2024-06-14T11:51:27.403+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:27 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users with view permissions: ***; groups with view permissions: EMPTY; users with modify permissions: ***; groups with modify permissions: EMPTY
[2024-06-14T11:51:27.823+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:27 INFO Utils: Successfully started service 'sparkDriver' on port 45851.
[2024-06-14T11:51:27.863+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:27 INFO SparkEnv: Registering MapOutputTracker
[2024-06-14T11:51:27.924+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:27 INFO SparkEnv: Registering BlockManagerMaster
[2024-06-14T11:51:27.952+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:27 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2024-06-14T11:51:27.953+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:27 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2024-06-14T11:51:27.957+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:27 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2024-06-14T11:51:28.010+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:28 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-6e93d2ab-1ff2-4de7-b835-399c6ade733b
[2024-06-14T11:51:28.034+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:28 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2024-06-14T11:51:28.051+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:28 INFO SparkEnv: Registering OutputCommitCoordinator
[2024-06-14T11:51:28.321+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:28 INFO JettyUtils: Start Jetty 0.0.0.0:4040 for SparkUI
[2024-06-14T11:51:28.418+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:28 INFO Utils: Successfully started service 'SparkUI' on port 4040.
[2024-06-14T11:51:28.660+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:28 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark-master:7077...
[2024-06-14T11:51:28.808+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:28 INFO TransportClientFactory: Successfully created connection to spark-master/172.18.0.3:7077 after 65 ms (0 ms spent in bootstraps)
[2024-06-14T11:51:29.255+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:29 INFO StandaloneSchedulerBackend: Connected to Spark cluster with app ID app-20240614115129-0008
[2024-06-14T11:51:29.273+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:29 INFO StandaloneAppClient$ClientEndpoint: Executor added: app-20240614115129-0008/0 on worker-20240614074744-172.18.0.5-36477 (172.18.0.5:36477) with 2 core(s)
[2024-06-14T11:51:29.281+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:29 INFO StandaloneSchedulerBackend: Granted executor ID app-20240614115129-0008/0 on hostPort 172.18.0.5:36477 with 2 core(s), 1024.0 MiB RAM
[2024-06-14T11:51:29.285+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:29 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 39143.
[2024-06-14T11:51:29.287+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:29 INFO NettyBlockTransferService: Server created on 963b3fbfa4c9:39143
[2024-06-14T11:51:29.291+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:29 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2024-06-14T11:51:29.303+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:29 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, 963b3fbfa4c9, 39143, None)
[2024-06-14T11:51:29.309+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:29 INFO BlockManagerMasterEndpoint: Registering block manager 963b3fbfa4c9:39143 with 434.4 MiB RAM, BlockManagerId(driver, 963b3fbfa4c9, 39143, None)
[2024-06-14T11:51:29.311+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:29 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, 963b3fbfa4c9, 39143, None)
[2024-06-14T11:51:29.312+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:29 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, 963b3fbfa4c9, 39143, None)
[2024-06-14T11:51:29.702+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:29 INFO StandaloneSchedulerBackend: SchedulerBackend is ready for scheduling beginning after reached minRegisteredResourcesRatio: 0.0
[2024-06-14T11:51:30.033+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:30 INFO StandaloneAppClient$ClientEndpoint: Executor updated: app-20240614115129-0008/0 is now RUNNING
[2024-06-14T11:51:30.234+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:30 INFO SharedState: Setting hive.metastore.warehouse.dir ('null') to the value of spark.sql.warehouse.dir.
[2024-06-14T11:51:30.249+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:30 INFO SharedState: Warehouse path is 'file:/opt/***/spark-warehouse'.
[2024-06-14T11:51:34.227+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:34 WARN FileStreamSink: Assume no metadata directory. Error while looking for metadata directory in the path: s3a://data-lake/vnindex.csv.
[2024-06-14T11:51:34.239+0000] {spark_submit.py:571} INFO - java.lang.RuntimeException: java.lang.ClassNotFoundException: Class org.apache.hadoop.fs.s3a.S3AFileSystem not found
[2024-06-14T11:51:34.242+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.conf.Configuration.getClass(Configuration.java:2688)
[2024-06-14T11:51:34.250+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.fs.FileSystem.getFileSystemClass(FileSystem.java:3431)
[2024-06-14T11:51:34.253+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.fs.FileSystem.createFileSystem(FileSystem.java:3466)
[2024-06-14T11:51:34.256+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.fs.FileSystem.access$300(FileSystem.java:174)
[2024-06-14T11:51:34.257+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.fs.FileSystem$Cache.getInternal(FileSystem.java:3574)
[2024-06-14T11:51:34.257+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.fs.FileSystem$Cache.get(FileSystem.java:3521)
[2024-06-14T11:51:34.259+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:540)
[2024-06-14T11:51:34.260+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.fs.Path.getFileSystem(Path.java:365)
[2024-06-14T11:51:34.264+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.execution.streaming.FileStreamSink$.hasMetadata(FileStreamSink.scala:53)
[2024-06-14T11:51:34.265+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.execution.datasources.DataSource.resolveRelation(DataSource.scala:366)
[2024-06-14T11:51:34.266+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.DataFrameReader.loadV1Source(DataFrameReader.scala:229)
[2024-06-14T11:51:34.267+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.DataFrameReader.$anonfun$load$2(DataFrameReader.scala:211)
[2024-06-14T11:51:34.267+0000] {spark_submit.py:571} INFO - at scala.Option.getOrElse(Option.scala:189)
[2024-06-14T11:51:34.268+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.DataFrameReader.load(DataFrameReader.scala:211)
[2024-06-14T11:51:34.269+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.DataFrameReader.csv(DataFrameReader.scala:538)
[2024-06-14T11:51:34.269+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[2024-06-14T11:51:34.272+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[2024-06-14T11:51:34.274+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[2024-06-14T11:51:34.275+0000] {spark_submit.py:571} INFO - at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[2024-06-14T11:51:34.277+0000] {spark_submit.py:571} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:244)
[2024-06-14T11:51:34.277+0000] {spark_submit.py:571} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:374)
[2024-06-14T11:51:34.278+0000] {spark_submit.py:571} INFO - at py4j.Gateway.invoke(Gateway.java:282)
[2024-06-14T11:51:34.279+0000] {spark_submit.py:571} INFO - at py4j.commands.AbstractCommand.invokeMethod(AbstractCommand.java:132)
[2024-06-14T11:51:34.281+0000] {spark_submit.py:571} INFO - at py4j.commands.CallCommand.execute(CallCommand.java:79)
[2024-06-14T11:51:34.284+0000] {spark_submit.py:571} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2024-06-14T11:51:34.285+0000] {spark_submit.py:571} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2024-06-14T11:51:34.286+0000] {spark_submit.py:571} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2024-06-14T11:51:34.287+0000] {spark_submit.py:571} INFO - Caused by: java.lang.ClassNotFoundException: Class org.apache.hadoop.fs.s3a.S3AFileSystem not found
[2024-06-14T11:51:34.288+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.conf.Configuration.getClassByName(Configuration.java:2592)
[2024-06-14T11:51:34.288+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.conf.Configuration.getClass(Configuration.java:2686)
[2024-06-14T11:51:34.290+0000] {spark_submit.py:571} INFO - ... 26 more
[2024-06-14T11:51:34.331+0000] {spark_submit.py:571} INFO - Traceback (most recent call last):
[2024-06-14T11:51:34.333+0000] {spark_submit.py:571} INFO - File "/opt/***/jobs/python/test2.py", line 10, in <module>
[2024-06-14T11:51:34.343+0000] {spark_submit.py:571} INFO - df = spark.read.csv(minio_url, header=True, inferSchema=True)
[2024-06-14T11:51:34.344+0000] {spark_submit.py:571} INFO - File "/home/***/.local/lib/python3.9/site-packages/pyspark/python/lib/pyspark.zip/pyspark/sql/readwriter.py", line 727, in csv
[2024-06-14T11:51:34.348+0000] {spark_submit.py:571} INFO - File "/home/***/.local/lib/python3.9/site-packages/pyspark/python/lib/py4j-0.10.9.7-src.zip/py4j/java_gateway.py", line 1322, in __call__
[2024-06-14T11:51:34.352+0000] {spark_submit.py:571} INFO - File "/home/***/.local/lib/python3.9/site-packages/pyspark/python/lib/pyspark.zip/pyspark/errors/exceptions/captured.py", line 169, in deco
[2024-06-14T11:51:34.356+0000] {spark_submit.py:571} INFO - File "/home/***/.local/lib/python3.9/site-packages/pyspark/python/lib/py4j-0.10.9.7-src.zip/py4j/protocol.py", line 326, in get_return_value
[2024-06-14T11:51:34.360+0000] {spark_submit.py:571} INFO - py4j.protocol.Py4JJavaError: An error occurred while calling o30.csv.
[2024-06-14T11:51:34.361+0000] {spark_submit.py:571} INFO - : java.lang.RuntimeException: java.lang.ClassNotFoundException: Class org.apache.hadoop.fs.s3a.S3AFileSystem not found
[2024-06-14T11:51:34.365+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.conf.Configuration.getClass(Configuration.java:2688)
[2024-06-14T11:51:34.371+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.fs.FileSystem.getFileSystemClass(FileSystem.java:3431)
[2024-06-14T11:51:34.380+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.fs.FileSystem.createFileSystem(FileSystem.java:3466)
[2024-06-14T11:51:34.385+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.fs.FileSystem.access$300(FileSystem.java:174)
[2024-06-14T11:51:34.389+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.fs.FileSystem$Cache.getInternal(FileSystem.java:3574)
[2024-06-14T11:51:34.395+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.fs.FileSystem$Cache.get(FileSystem.java:3521)
[2024-06-14T11:51:34.407+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.fs.FileSystem.get(FileSystem.java:540)
[2024-06-14T11:51:34.413+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.fs.Path.getFileSystem(Path.java:365)
[2024-06-14T11:51:34.417+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.execution.datasources.DataSource$.$anonfun$checkAndGlobPathIfNecessary$1(DataSource.scala:724)
[2024-06-14T11:51:34.425+0000] {spark_submit.py:571} INFO - at scala.collection.immutable.List.map(List.scala:293)
[2024-06-14T11:51:34.431+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.execution.datasources.DataSource$.checkAndGlobPathIfNecessary(DataSource.scala:722)
[2024-06-14T11:51:34.436+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.execution.datasources.DataSource.checkAndGlobPathIfNecessary(DataSource.scala:551)
[2024-06-14T11:51:34.439+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.execution.datasources.DataSource.resolveRelation(DataSource.scala:404)
[2024-06-14T11:51:34.441+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.DataFrameReader.loadV1Source(DataFrameReader.scala:229)
[2024-06-14T11:51:34.444+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.DataFrameReader.$anonfun$load$2(DataFrameReader.scala:211)
[2024-06-14T11:51:34.448+0000] {spark_submit.py:571} INFO - at scala.Option.getOrElse(Option.scala:189)
[2024-06-14T11:51:34.449+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.DataFrameReader.load(DataFrameReader.scala:211)
[2024-06-14T11:51:34.449+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.DataFrameReader.csv(DataFrameReader.scala:538)
[2024-06-14T11:51:34.451+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[2024-06-14T11:51:34.452+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[2024-06-14T11:51:34.453+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[2024-06-14T11:51:34.454+0000] {spark_submit.py:571} INFO - at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[2024-06-14T11:51:34.455+0000] {spark_submit.py:571} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:244)
[2024-06-14T11:51:34.457+0000] {spark_submit.py:571} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:374)
[2024-06-14T11:51:34.458+0000] {spark_submit.py:571} INFO - at py4j.Gateway.invoke(Gateway.java:282)
[2024-06-14T11:51:34.459+0000] {spark_submit.py:571} INFO - at py4j.commands.AbstractCommand.invokeMethod(AbstractCommand.java:132)
[2024-06-14T11:51:34.461+0000] {spark_submit.py:571} INFO - at py4j.commands.CallCommand.execute(CallCommand.java:79)
[2024-06-14T11:51:34.463+0000] {spark_submit.py:571} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2024-06-14T11:51:34.465+0000] {spark_submit.py:571} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2024-06-14T11:51:34.465+0000] {spark_submit.py:571} INFO - at java.base/java.lang.Thread.run(Thread.java:829)
[2024-06-14T11:51:34.466+0000] {spark_submit.py:571} INFO - Caused by: java.lang.ClassNotFoundException: Class org.apache.hadoop.fs.s3a.S3AFileSystem not found
[2024-06-14T11:51:34.467+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.conf.Configuration.getClassByName(Configuration.java:2592)
[2024-06-14T11:51:34.469+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.conf.Configuration.getClass(Configuration.java:2686)
[2024-06-14T11:51:34.469+0000] {spark_submit.py:571} INFO - ... 29 more
[2024-06-14T11:51:34.470+0000] {spark_submit.py:571} INFO - 
[2024-06-14T11:51:34.684+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:34 INFO SparkContext: Invoking stop() from shutdown hook
[2024-06-14T11:51:34.686+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:34 INFO SparkContext: SparkContext is stopping with exitCode 0.
[2024-06-14T11:51:34.713+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:34 INFO SparkUI: Stopped Spark web UI at http://963b3fbfa4c9:4040
[2024-06-14T11:51:34.723+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:34 INFO StandaloneSchedulerBackend: Shutting down all executors
[2024-06-14T11:51:34.729+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:34 INFO StandaloneSchedulerBackend$StandaloneDriverEndpoint: Asking each executor to shut down
[2024-06-14T11:51:34.766+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:34 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2024-06-14T11:51:34.811+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:34 INFO MemoryStore: MemoryStore cleared
[2024-06-14T11:51:34.814+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:34 INFO BlockManager: BlockManager stopped
[2024-06-14T11:51:34.825+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:34 INFO BlockManagerMaster: BlockManagerMaster stopped
[2024-06-14T11:51:34.837+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:34 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2024-06-14T11:51:34.868+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:34 INFO SparkContext: Successfully stopped SparkContext
[2024-06-14T11:51:34.869+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:34 INFO ShutdownHookManager: Shutdown hook called
[2024-06-14T11:51:34.872+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:34 INFO ShutdownHookManager: Deleting directory /tmp/spark-3c7627d2-83c6-4933-a3cd-4ef68078974c
[2024-06-14T11:51:34.886+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:34 INFO ShutdownHookManager: Deleting directory /tmp/spark-3c7627d2-83c6-4933-a3cd-4ef68078974c/pyspark-28e2d995-6d30-4e07-8822-9bd76de554ba
[2024-06-14T11:51:34.901+0000] {spark_submit.py:571} INFO - 24/06/14 11:51:34 INFO ShutdownHookManager: Deleting directory /tmp/spark-cb40ef45-21ad-460a-bcb2-1cabc0e2e533
[2024-06-14T11:51:35.257+0000] {taskinstance.py:1935} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 174, in execute
    self._hook.submit(self.application)
  File "/home/airflow/.local/lib/python3.9/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 502, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark-master:7077 --conf spark.hadoop.fs.s3a.endpoint=http://host.docker.internal:9000 --conf spark.hadoop.fs.s3a.access.key=khaiVDT2024 --conf spark.hadoop.fs.s3a.secret.key=****** --conf spark.hadoop.fs.s3a.path.style.access=true --name spark_minio_job --deploy-mode client jobs/python/test2.py. Error code is: 1.
[2024-06-14T11:51:35.302+0000] {taskinstance.py:1398} INFO - Marking task as FAILED. dag_id=simple_spark_dag, task_id=spark_submit_task, execution_date=20240614T115113, start_date=20240614T115117, end_date=20240614T115135
[2024-06-14T11:51:35.384+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 69 for task spark_submit_task (Cannot execute: spark-submit --master spark://spark-master:7077 --conf spark.hadoop.fs.s3a.endpoint=http://host.docker.internal:9000 --conf spark.hadoop.fs.s3a.access.key=khaiVDT2024 --conf spark.hadoop.fs.s3a.secret.key=****** --conf spark.hadoop.fs.s3a.path.style.access=true --name spark_minio_job --deploy-mode client jobs/python/test2.py. Error code is: 1.; 9741)
[2024-06-14T11:51:35.520+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2024-06-14T11:51:35.684+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
